spring/boot3-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSettingsConfigurationTest.java

```diff
@@ -38,7 +38,7 @@
 @SpringBootTest(classes = TestConfiguration.class)
 @ActiveProfiles({ "local", "settings" })
 @DirtiesContext
-class ArmeriaSettingsConfigurationTest {
+public class ArmeriaSettingsConfigurationTest {
```

Comment:
Question: Why was this necessary?

Reply:
`ArmeriaSettingsConfigurationTest` uses JUnit 4 which requires a public class.
I don't get why it wasn't detected before.